mysql-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mysql-operator
-version: 0.2.0
+version: 0.2.1
```

Comment:
This is a catch 22 change where we need a release created or master will be broken for a while.

Reply:
The chart version is independent of the release version.

Reply:
Just realised and was about to undo the comment. Thanks.

Reply:
My mistake. This is actually fine.